.github/workflows/conan.yml

```diff
@@ -27,6 +27,13 @@ jobs:
           sudo mkdir build && cd build
           sudo cmake .. && sudo make
           sudo apt install libgmock-dev -y
+
+      - name: Install newer Clang
+        run: |
+          wget https://apt.llvm.org/llvm.sh
+          chmod +x ./llvm.sh
+          sudo ./llvm.sh 17
```

Comment:
We should do one thing in one patch. Please move this to other PR.

Reply:
OK, when test passed will create a new pr to upgrade clang for conan.